src/Components/AddTask/AddTask.tsx

```diff
@@ -1,19 +1,20 @@
-// nadia.mohamed.taha166@gmail.com
-// @Password123!
+
 import axios from "axios";
 import { useContext, useEffect, useState } from "react";
 import { useNavigate } from "react-router-dom";
 import { AuthContext } from "./../../Context/AuthContext";
 import { ToastContext } from "../../Context/ToastContext";
 import { SubmitHandler, useForm } from "react-hook-form";
 import { ProjectContext } from '../../Context/ProjectContext';
-import { FormValues, projectType } from './../../Types/Types';
+import { FormValues, projectType ,IAuth} from './../../Types/Types';
 
 
 const AddTask: React.FC = () => {
-  const { baseUrl, requestHeaders }: any = useContext(AuthContext);
-  const { getToastValue }: any = useContext(ToastContext);
-  const {  projects }: any = useContext(ProjectContext);
+  const { baseUrl, requestHeaders }:any
+  //Pick<IAuth, 'baseUrl','requestHeaders'> 
+  = useContext(AuthContext);
+  const { getToastValue }: void = useContext(ToastContext);
+  const {  projects }: projectType = useContext(ProjectContext);
```

Comment:
The "projects" is an array containing elements of the "projectType" type. Therefore, it is recommended to declare the type as an array, specifying it as "projectType[]".